app/containers/AccountProvider/saga.js

```diff
@@ -219,6 +231,10 @@ function* updateRefer(user, eosService) {
     setCookie({
       name: noInviterCookieName,
       value: true,
+      options: {
```

Comment:
The same options are passed to `setCookie `. Might be better to pass flag arguments to that function and then let function to set options. E.g. `neverExpires`, `allowSubdomains` boolean arguments.